src/sentry/models/authenticator.py

```diff
@@ -1,204 +1,3 @@
-from __future__ import annotations
+from ..users.models.auth.authenticator import Authenticator  # NOQA
```

Comment:
Could we do `from sentry.users.models.authenticator import Authenticator`. The users app will have fewer than 10 models in it and I don't think the additional layer of directories adds much.